recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+
+[Service]
+User=root
+EnvironmentFile=-/etc/default/weston
+ExecStartPre=/usr/bin/weston-waitforcard.sh
```

Comment:
That didn't work out (dev-dri-card0.path). As a device that didn't work either (dev-dri-card0.device). As ConditionPathExists in the service unit, it failed as well.

Another option was to simply let Weston fail (due to drm) and set RestartSec=30 with Restart=on-failure. I didn't like _that_ approach, but also worked.